fs/tmpfs/fs_tmpfs.c

```diff
@@ -1442,7 +1442,11 @@ static int tmpfs_close(FAR struct file *filep)
        * have any other references.
        */
 
-      kmm_free(tfo->tfo_data);
```

Comment:
but kmm_free/mm_free will check NULL for us.

Reply:
It is this line that is causing the assertion violation: [kmm_free.c:55](https://github.com/apache/incubator-nuttx/blob/77a99acc98737a9e38694fa8270de806b4087e53/mm/kmm_heap/kmm_free.c#L55).  Perhaps it should be`DEBUGASSERT((mem == NULL) || kmm_heapmember(mem));`?

Reply:
Yes, I think so. or it's better to move the assert to mm_free.